code/modules/vending/clothesmate.dm

```diff
@@ -77,6 +77,7 @@
 		/obj/item/clothing/suit/jacket = 2,
 		//obj/item/clothing/suit/hooded/wintercoat/custom = 2, SKYRAT EDIT - Moved to modular equivalent
 		/obj/item/clothing/suit/jacket/puffer/vest = 2,
+		/obj/item/clothing/suit/pink_jacket = 3,
```

Comment:
move this to below this line here,
https://github.com/Skyrat-SS13/Skyrat-tg/blob/10a2c07de6e0b2d7fa20a40f70471d730f820563/modular_skyrat/modules/modular_vending/code/clothesmate.dm#L95